charts/nri-kube-events/values.yaml

```diff
@@ -36,6 +36,29 @@ sinks:
   # The newRelicInfra sink sends all events to New relic.
   newRelicInfra: true
 
+# If you wish to provide your own newrelic.yml file include it under config:
+# the sample config file is included here as an example. Note that some option
+# overlap with environment variables: for example proxy
+# config:
+  #
+  # New Relic Infrastructure configuration file
+  # https://github.com/newrelic/infrastructure-agent/blob/master/assets/examples/infrastructure/newrelic-infra-template.yml.example
+  #
+  # Lines that begin with # are comment lines and are ignored by the
+  # Infrastructure agent. If options have command line equivalents, New Relic
+  # will use the command line option to override any value set in this file.
+  #
+  #
+  # You can use this section in order to add any agent configuration.
+  # For example to add FedRAMP-compliant endpoints(https://docs.newrelic.com/docs/security/security-privacy/compliance/fedramp-compliant-endpoints#infrastructure-endpoints),
+  # to set verbose logs, etc
+  # verbose: 1
+  # FedRAMP
+  # collector_url: https://gov-infra-api.newrelic.com
```

Comment:
I think we can avoid using this particular example.

Reply:
I added them since they are not listed in the standard documentation page, but only in this doc:
https://docs.newrelic.com/docs/security/security-privacy/compliance/fedramp-compliant-endpoints#infrastructure-endpoints

But you are right it might be too specific